### ISSUE TITLE
PHP 7 syntax required

### ISSUE THREAD

**Issue by mr0bles** [renatomarinho/laravel-page-speed#11]
When i install it on Laravel 5.3 with PHP 5.6 i have this problem
```
FatalErrorException in PageSpeed.php line 26:
syntax error, unexpected ':', expecting ';' or '{'
```

This is a problem of PageSpeed that required a syntax of PHP 7

**Comment by joaorobertopb** [comment]
Hi, 

This problem was referenced in issue #9 and fixed in PR #10. ( By now use dev-master in composer )

@renatomarinho Please add new tag 1.5.1 with this corrections.

**Comment by renatomarinho** [comment]
done
